CONTRIBUTING.md

```diff
@@ -44,22 +45,39 @@ We recommend IntelliJ IDEA Community, because all of its plugins and
 configurations possibilities, here's [the website](https://www.jetbrains.com/idea/download).
 But feel free to use Eclipse or other IDE of your choice.
 
-## Follow our best practices
+### Check-style
```

Comment:
There is an extra '#' here, it can be removed to remain in the same pattern as the others.

Reply:
Actually, is correct. The "Check-style" item is nested with "Set up your environment". So three is the right one.